meson.build

```diff
@@ -204,6 +204,9 @@ if get_option('build_backends')
         opencl_files += 'src/neural/blas/transforms.cc'
       endif
 
+      deps += dependency('boost', required: true)
```

Comment:
This will need to be platform dependent, since boost is not required on Linux

Reply:
It was required, but only some headers and they were already in /usr/include so the build worked. However the latest version removes the boost dependency completely.